dotcom-rendering/src/web/components/StarRating/StarRating.tsx

```diff
@@ -36,27 +37,46 @@ const determineSize = (size: RatingSizeType) => {
 	}
 };
 
+const determineBreakpoint = (breakpoint: Breakpoint | undefined) =>
+	breakpoint !== undefined ? breakpoint : '';
```

Comment:
I wonder if we could have the default string be `'unset-breakpoint'`, `'all-breakpoints'` or similar? Just so if you were looking at the DOM for those cases the starId might make more sense?